doc/whats-new.rst

```diff
@@ -25,6 +25,13 @@ Enhancements
   raster files are opened with :py:func:`~xarray.open_rasterio`.
   By `Greg Brener <https://github.com/gbrener>`_
 
+- A new resampling interface to match pandas' group-by-like API was added to
```

Comment:
After you merge in master, please move this into the "Breaking changes" section.